es/installation/docker.md

```diff
@@ -1,7 +1,7 @@
 # Usando Docker para desarrollo en local
 
 Puedes usar Docker para tener una instalación local de CONSUL si:
-- Estás teniendo problemas para instalar los [prerrequisitos](prerequisites) correctamente.
+- Estás teniendo problemas para instalar los [prerrequisitos](prerequisites.md) correctamente.
```

Comment:
Correct me if I'm wrong; I thought the link was supposed to point to ../local_installation/prerequisites :thinking:

Reply:
It's the same case: https://github.com/consul/docs/pull/89#discussion_r383174998